infrastructure/cdn-in-a-box/traffic_ops/Dockerfile-go

```diff
@@ -45,18 +45,18 @@ FROM    trafficops-common-deps
 
 # Override TRAFFIC_OPS_RPM arg to use a different one using --build-arg TRAFFIC_OPS_RPM=...  Can be local file or http://...
 #
-ARG     TRAFFIC_OPS_RPM=infrastructure/cdn-in-a-box/traffic_ops/traffic_ops.rpm
-
-COPY    $TRAFFIC_OPS_RPM /
-RUN     to_rpm_filename="$(basename $TRAFFIC_OPS_RPM)" && \
-        rpm --install --nodeps --verbose --hash "$to_rpm_filename" && \
-        rm "$to_rpm_filename"
-
-COPY    infrastructure/cdn-in-a-box/enroller/server_template.json \
-        infrastructure/cdn-in-a-box/traffic_ops/config.sh \
-        infrastructure/cdn-in-a-box/traffic_ops/run-go.sh \
-        infrastructure/cdn-in-a-box/traffic_ops/to-access.sh \
-        /
+ARG TRAFFIC_OPS_RPM=traffic_ops/traffic_ops.rpm
+
+COPY $TRAFFIC_OPS_RPM /
+RUN to_rpm_filename="$(basename $TRAFFIC_OPS_RPM)" && \
+    rpm --install --nodeps --verbose --hash "$to_rpm_filename" && \
+    rm "$to_rpm_filename"
+
+COPY enroller/server_template.json \
+     traffic_ops/config.sh \
+     traffic_ops/run-go.sh \
+     traffic_ops/to-access.sh \
+     /
```

Comment:
These path changes need to be made in `/infrastructure/cdn-in-a-box/traffic_ops/Dockerfile-go.dockerignore` for any Docker BuildKit users